controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3332,35 +3333,17 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
     @Message(id = 398, value = "An attempt was made to rename the resource found at %s to %s. However, '%s' is one of the resource types defined to be ordered on the parent resource %s")
     OperationFailedRuntimeException orderedChildTypeRenamed(PathAddress read, PathAddress transformed, String type, Set<String> parentOrderedChildren);
 
-    @Message(id = 399, value = "Model references of type '%s' are missing: %s")
-    OperationFailedException missingReferences(String type, Set<String> missing);
```

Comment:
Did we ship any of these ids in WildFly 9? If so, they can't be repurposed and should just be commented out if no longer used.